packaging/Flatpak/org.telesculptor.TeleSculptor.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Name=TeleSculptor
+GenericName=Aerial Photogrammetry
+Comment=Photogrammetry designed with a focus on aerial video
+Exec=TeleSculptor %U
+Terminal=false
+Type=Application
+Icon=org.telesculptor.TeleSculptor
+Categories=Qt;Science;DataVisualization;
```

Comment:
We might consider adding `ImageProcessing`, `Geoscience`, and `3DGraphics`, maybe also `AudioVideo;Video`.  There are other categories that might fit too.

Reply:
Does TeleSculptor also have audio, so it can be include in AudioVideo category?

Reply:
Yeah, no audio, but it sounds like to include `Video` you also need `AudioVideo` as a super category.